rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -1675,26 +1675,12 @@ void spu_thread::cpu_task()
 	}
 }
 
-spu_thread::~spu_thread()
+void spu_thread::cleanup()
 {
-	{
-		vm::writer_lock(0);
```

Comment:
Had no name, destructor called immediately.

Reply:
Did you fix the constructor too? Damn copypaste.

Reply:
Fixed now.